js/main.js

```diff
@@ -1,13 +1,223 @@
 /* global _ */
-var content = "";
-var error = "";
 var xhr = new XMLHttpRequest();
 
-// construct a quote
+var singularize = function(word) {
+    word = word.trim();
+    if (word.substr(-1) === 's') {
+        return word.slice(0, -1);
+    } else {
+        return word;
+    }
+};
```

Comment:
Hah, I was mostly joking, but it turns out there is a pretty good looking one: https://www.npmjs.com/package/underscore.inflections